src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,71 @@
+package duke.command;
+
+import duke.data.Storage;
+import duke.data.TaskList;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.TodoTask;
+import duke.util.DukeException;
+import duke.util.Ui;
+
+/**
+ * Executes the command to add a task to the list.
+ * @author Jicson Toh
+ */
+public class AddCommand extends Command {
+    private final String action;
```

Comment:
Should constant names be in lower case? Some other constant names have the same issue too.

Reply:
hmm its not a constant but a variable to be instantiated when creating the AddCommand Object